build.sbt

```diff
@@ -128,71 +143,24 @@ lazy val examples = project
 
 lazy val site = project
   .in(file("site"))
-  .disablePlugins(MimaPlugin)
-  .enablePlugins(MicrositesPlugin)
-  .enablePlugins(MdocPlugin)
-  .enablePlugins(NoPublishPlugin)
-  .settings(commonSettings)
+  .settings(publish / skip := true)
+  .enablePlugins(TypelevelSitePlugin)
+  .enablePlugins(TypelevelUnidocPlugin)
   .dependsOn(core, avro4s)
   .settings {
-    import microsites._
     Seq(
-      micrositeName := "kafka4s",
-      micrositeDescription := "Functional programming with Kafka and Scala",
-      micrositeAuthor := "Jack Henry & Associates, Inc.®",
```

Comment:
Are these values retained by some other mechanism, or just lost? I presume some of it can be inferred from the new plugins, but all of it even this author field?

Reply:
Fair question.  I've tried in 5f85719.